### DISCUSSION TITLE
2022/12/08/LIFE-OF-A-STUDENT(EXPECTATION-VS-REALITY)

### DISCUSSION THREAD

**Discussion by giscus** [KMIT-Students/blog-demo#14] [General]
# 2022/12/08/LIFE-OF-A-STUDENT(EXPECTATION-VS-REALITY)

A student thinks in a very positive way when he/she joins a college but the reality is way too harsh. When the student is pursuing his/her intermediate, every student dreams that he/she will join the IIT or NIT or at least IIIT but more than 95% of students join EAMCET colleges. They see the lecture videos of famous and well-educated teachers ALAKH PANDEY,NISHANT VORA and dream that they perfected the chapter and they can crack JEE but once they start to solve the questions their fuses go off. Finally, when they appear for exams, they get tensed and depressed. Every student minimum appears for 5-7 exams for joining Btech colleges. Finally, they join the EAMCET college. Here the scene goes completely differently. Once one opens the homepage of the college, he/she will see the photos of other students who have secured 40+L package and they start dreaming that they would secure 40l+ package. They enquire about the college and get hyped about the college. But once they enter college, they feel that they regret the decision of joining the college. A maximum of people see the movies and get overjoyed that they’re going to enjoy their 4 years of life but they don’t recognize that they need to run for placement’s sake. College management says that they are gonna organize many events, seminars, hackathons, sports competitions, etc but the reality is labs, assignments, notes, and scoldings. And the main thing is that most of the teenage student thinks when he/she enters college they can have a crush but nothing happens you just see and feel happy yourselves. The only thing left for you when you join the college is to study study study. Finally, in my opinion, the student should never get frightened about his/her future in just only 1st year and enjoy the college and be happy and take good memories

https://kmit-anubhav.github.io/2022/12/08/LIFE-OF-A-STUDENT(EXPECTATION-VS-REALITY).html

**Comment by creator318** [comment]
An accurate comparison.
Very real....
